code/addons/toolbars/src/components/ToolbarMenuList.tsx

```diff
@@ -79,6 +82,7 @@ export const ToolbarMenuList: FC<ToolbarMenuListProps> = withKeyboardCycle(
       >
         <ToolbarMenuButton
           active={isTooltipVisible || hasGlobalValue}
+          disabled={isOverridden}
```

Comment:
Readonly might be a better UX. See this recent PR

https://github.com/storybookjs/storybook/pull/26577

Reply:
Can we leave it to someone else (w/ @cdedreuille's help) to create a readonly version of the `TooltipLinkList` & `ToolbarMenuButton`? They doesn't appear to exist at the moment.